### PR TITLE
docs(es): Translate Non-standardGeneric macro to spanish[es]

### PR DESCRIPTION
**Pull request by davbrito**

### CODE REVIEW

**Review by Vallejoanderson — CHANGES_REQUESTED** [review]

File: kumascript/macros/Non-standardGeneric.ejs [hunk 1]
```diff
@@ -17,7 +17,8 @@ var str_inline = mdn.localString({
 
 var str_long = mdn.localString({
   "en-US": "This feature is non-standard and is not on a standards track. Do not use it on production sites facing the Web: it will not work for every user. There may also be large incompatibilities between implementations and the behavior may change in the future.",
-  "de":    "Diese Funktion entspricht nicht dem Standard und ist nicht Teil der Standardisierung. Diese Funktion darf nicht in Webseiten, die via das Internet zugänglich sind, benutzt werden: Sie wird nicht für alle Nutzer funktionieren. Es kann zu umfangreichen Inkompatibilitäten zwischen verschiedenen Implementierungen kommen und die Funktionsweise oder Eigenschaften könnten in der Zukunft verändert werden.",
+  "de":    "Diese Funktion entspricht nicht dem Standard und ist nicht Teil der Standardisierung. Diese Funktion darf nicht in Webseiten, die via das Internet zugänglich sind, benutzt werden: Sie wird nicht für alle Nutzer funktionieren. Es kann zu umfangreichen Inkompatibilitäten zwischen verschiedenen Implementierungen kommen und die Funktionsweise oder Eigenschaften könnten in der Zukunft verändert werden.",,
```
Comment (schalkneethling) [inline kumascript/macros/Non-standardGeneric.ejs:20]:
```suggestion
  "de":    "Diese Funktion entspricht nicht dem Standard und ist nicht Teil der Standardisierung. Diese Funktion darf nicht in Webseiten, die via das Internet zugänglich sind, benutzt werden: Sie wird nicht für alle Nutzer funktionieren. Es kann zu umfangreichen Inkompatibilitäten zwischen verschiedenen Implementierungen kommen und die Funktionsweise oder Eigenschaften könnten in der Zukunft verändert werden.",
```

**Review by schalkneethling — CHANGES_REQUESTED** [review]
Hey @davbrito! Thank you for your contribution. Just one small typo and this should be good to go.

File: kumascript/macros/Non-standardGeneric.ejs [hunk 2]
```diff
@@ -18,6 +18,7 @@ var str_inline = mdn.localString({
 var str_long = mdn.localString({
   "en-US": "This feature is non-standard and is not on a standards track. Do not use it on production sites facing the Web: it will not work for every user. There may also be large incompatibilities between implementations and the behavior may change in the future.",
   "de":    "Diese Funktion entspricht nicht dem Standard und ist nicht Teil der Standardisierung. Diese Funktion darf nicht in Webseiten, die via das Internet zugänglich sind, benutzt werden: Sie wird nicht für alle Nutzer funktionieren. Es kann zu umfangreichen Inkompatibilitäten zwischen verschiedenen Implementierungen kommen und die Funktionsweise oder Eigenschaften könnten in der Zukunft verändert werden.",
+  "es":    "Esta característica no es parte de los estándares. No la uses en sitios Web en producción: no funcionará para todos los usuarios. Podrían haber también incompatibilidades considerables entre distintas implementaciones y el comportamiento podría cambiar en el futuro."
```
Comment (schalkneethling) [inline kumascript/macros/Non-standardGeneric.ejs:21]: Oops, looks like you accidentally deleted both commas. 

```suggestion
  "es":    "Esta característica no es parte de los estándares. No la uses en sitios Web en producción: no funcionará para todos los usuarios. Podrían haber también incompatibilidades considerables entre distintas implementaciones y el comportamiento podría cambiar en el futuro.",
```

Comment (schalkneethling) [reply kumascript/macros/Non-standardGeneric.ejs:21]: I am going to go ahead and commit the above suggestions. Thanks, @davbrito

**Review by schalkneethling — APPROVED** [review]

**Pull request merged**